Error Build Yuzu 271 in Linux

When I'm compiling this Build on Linux I get this problem:

vk_shader_decompiler.cpp:840:21: error: converstion from  ‘int’ to  ‘u8’
[-Werror=conversion]  element += static_cast<u8>(num_components);

In previous builds I did not have this problem.

I'm using Arch Linux ,

--- a/src/video_core/renderer_vulkan/vk_shader_decompiler.cpp
+++ b/src/video_core/renderer_vulkan/vk_shader_decompiler.cpp
@@ -837,7 +837,7 @@ private:
                 Decorate(id, spv::Decoration::Offset, static_cast<u32>(tfb->offset));
             }
 
-            element += static_cast<u8>(num_components);
+            element = static_cast<u8>(static_cast<std::size_t>(element) + num_components);
         }
     }
 

